tests/test_bgp_speaker.py

```diff
@@ -28,46 +27,44 @@ def generate_ips(num, prefix, exclude_ips):
 
     return generated_ips
 
+
 def announce_route(ptfip, neighbor, route, nexthop, port):
-    url  = "http://%s:%d" % (ptfip, port)
-    data = { "command": "neighbor %s announce route %s next-hop %s" % (neighbor, route, nexthop) }
+    url = "http://%s:%d" % (ptfip, port)
+    data = {"command": "neighbor %s announce route %s next-hop %s" % (neighbor, route, nexthop)}
     r = requests.post(url, data=data)
     assert r.status_code == 200
 
-@pytest.mark.parametrize(
-    "ipv4, ipv6, mtu",
-    [	pytest.param(True, False, 1514),	],
-    )
-def test_bgp(localhost, ansible_adhoc, testbed, ipv4, ipv6, mtu):
-    """setup bgp speaker on T0 topology and verify routes advertised
-    by bgp speaker is received by T0 TOR
-    """
 
-    hostname = testbed['dut']
-    ptf_hostname = testbed['ptf']
-    host = AnsibleHost(ansible_adhoc, hostname)
-    ptfhost = AnsibleHost(ansible_adhoc, ptf_hostname)
-    ptfip = ptfhost.host.options['inventory_manager'].get_host(ptf_hostname).vars['ansible_host']
+@pytest.fixture(scope="module")
+def common_setup_teardown(duthost, ptfhost):
+
+    logging.info("########### Setup for bgp speaker testing ###########")
```

Comment:
Could you provide an example of how to set log level

Reply:
We can use the pytest CLI argument to set log level, for example: `--log-cli-level debug`